setup.cfg

```diff
@@ -19,7 +19,7 @@ packages = find:
 python_requires = >=3.8
 install_requires =
     aiohttp
-    aiostream
```

Comment:
This shouldn't trigger image rebuilds right @mwaskom ? Afaicr only the modal/requirements.txt file does that 🤔

Reply:
That’s my understanding!